src/containers/Transactions/index.tsx

```diff
@@ -3,101 +3,160 @@
 import React, { useState } from 'react';
 import Image from 'next/image';
 
+import fetch from 'src/utils/request';
 import CCard from 'src/components/CCard';
+import { useAppSelector } from 'src/hooks/useRedux';
 import CStreamType from 'src/components/CStreamType';
-import { formatNumber } from 'src/utils/formatNumber';
+import humanizeAmount from 'src/utils/humanizeAmount';
 import capitalize from 'src/utils/capitalizeFirstLetter';
 import { shortenAddress } from 'src/utils/shortenAddress';
+import { findTokenByAddress } from 'src/utils/findTokenByAddress';
 import CStreamStatus, { StreamStatus } from 'src/components/CStreamStatus';
 
-import Funnel from 'src/assets/Funnel';
+// import Funnel from 'src/assets/Funnel';
 import divider from 'public/images/divider.svg';
+import noStreams from 'public/images/noStreams.svg';
 
-import usdc from 'public/images/usdc.svg';
-import MagnifyingGlass from 'src/assets/MagnifyingGlass';
+import usdc from 'public/images/assets/fusdc.svg';
+import searchLogo from 'public/images/search.svg';
 
 import * as Styled from './styles';
 import useFetchHistory from './useFetchHistory';
 import getStatusStyles from './getStatusStyle';
+import { ExternalPages } from 'src/constants/externalPages';
+import { useRouter } from 'next/navigation';
 
 const Transactions = () => {
   const [selectedStatus, setSelectedStatus] = useState<StreamStatus>(StreamStatus.ONGOING);
+  const [openSearch, setOpenSearch] = useState(false);
+  const [searchValue, setSearchValue] = useState('');
 
-  const address = 'GA3A24K44D5JXIJ4RDPZTZLGZCUCJTMO2HKCFJ5CK6FYTEVUEIICSIXW';
-
+  const router = useRouter();
+  const address = useAppSelector((state) => state.user.address);
+  const tokens = useAppSelector((state) => state.tokens);
   const streams = useFetchHistory(address);
 
-  const filteredStreams = selectedStatus
-    ? streams.filter((stream) => stream.streamStatus === selectedStatus)
+  const onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchValue(e.target.value);
+  };
+
+  const handleSearch = () => {
+    setOpenSearch(!openSearch);
+  };
+
+  const handleClick = (streamId: string) => {
+    router.push(`/stream/${streamId}`);
+  };
+
+  const filteredStreamsByStatus = selectedStatus
```

Comment:
selectedStatus is always truthy, because it has a default value "ongoing", so remove the `selectedStatus ? ..` check